staff_features/shared/step_definitions.rb

```diff
@@ -54,6 +80,12 @@
   expect(find('.alert.alert-success.with-hide-alert').text).to match(/^#{string}.*updated$/i)
 end
 
+Then('the following info message is displayed') do |messages|
```

Comment:
s/b 'the following info messages are displayed' because it accepts and tests multiple

Reply:
Fixed. There is a check for messages length now.